dss-api.yml

```diff
@@ -1431,6 +1431,23 @@ paths:
             type: object
             properties:
               files:
+                description: >
+                  The list of files making up the contents of the new bundle.  This is a list of dictionaries, each one
+                  describing the file, required, and including:
```

Comment:
This sentence is awkward, maybe try something like this.
"This is a list of dictionaries describing each of the files. Each dictionary includes the fields:"

Reply:
Done.